support/client/lib/vwf/view/blockly.js

```diff
@@ -218,34 +225,45 @@ define( [ "module", "vwf/view", "jquery", "vwf/model/blockly/JS-Interpreter/acor
                 var app = this.state.scenes[ nodeID ];
                 switch ( propertyName ) {
                     case "blockly_activeNodeID":
-                        if ( this.state.nodes[ propertyValue ] !== undefined ) {
+
+                        var newActiveNodeId = propertyValue;
+                        var previousActiveNode = this.state.blockly.node;
+                        var blocklyNode = this.state.nodes[ newActiveNodeId ];
+                        if ( blocklyNode !== undefined ) {
                             var show = true;
-                            node = this.state.nodes[ propertyValue ];
-                            if ( this.state.blockly.node !== undefined ) {
-                                getBlockXML( this.state.blockly.node );
-                                setBlocklyUIVisibility( this.state.blockly.node, false ); 
-                                show = ( this.state.blockly.node.ID !== propertyValue );
-                                this.state.blockly.node = undefined;                           
-                            } 
+
+                            //HACK: REMOVED THIS CODE - CAUSED REPLICATED BLOCKLY WINDOW TO CLOSE
+
+                            // If there was already an active blockly node, deal with it before
+                            // activating the new one
+                            //if ( previousActiveNode !== undefined ) {
+                             //   getBlockXML( previousActiveNode );
+                             //   setBlocklyUIVisibility( previousActiveNode, false ); 
+                             //   show = ( previousActiveNode.ID !== newActiveNodeId );
+                             //   this.state.blockly.node = undefined;                           
+                            //}
```

Comment:
I believe this is here to allow there to be more than one node in the scene that is controlled by blockly (like the rover and mini-rover in mars-game).  This property-change handler probably needs a check at the top to see if the new value is equal to the old, and if so, to exit early.  At that point, it would probably be fine to uncomment out this block.